scripts/router.js

```diff
@@ -0,0 +1,15 @@
+//Manipulate Pages SPA-Style
+page.base('');
+
+page('/', about);
+page('/projects', projects)
+
+page();
+
+function about() {
```

Comment:
That's fair, I could have done that as well. I was expecting there to be more code in the about() function, so I separated it incase I needed to declare anything else.